renderer/src/pages/dashboard/Dashboard.tsx

```diff
@@ -1,11 +1,11 @@
 import { useState } from 'react'
-import HeaderBackgroundImage from '../assets/img/header.png'
-import Modal from '../components/Modal'
-import ActivityLog from '../components/ActivityLog'
-import UpdateBanner from '../components/UpdateBanner'
-import WalletWidget from '../components/WalletWidget'
-import useStationActivity from '../hooks/StationActivity'
-import { formatTokenValue } from '../utils/number-ops'
+import HeaderBackgroundImage from '../../assets/img/header.png'
+import Modal from '../../components/Modal'
+import ActivityLog from '../../components/ActivityLog'
+import UpdateBanner from '../../components/UpdateBanner'
+import WalletWidget from '../../components/WalletWidget'
+import useStationActivity from '../../hooks/StationActivity'
+import { formatTokenValue } from '../../utils/number-ops'
```

Comment:
@juliangruber it would simplify things if we updated `tsconfig` to use absolute paths for imports instead of relative. I can add this in a follow up PR, or add it here if you think it makes sense.

Reply:
It's here: https://github.com/subvisual/desktop/pull/1